.ci/pipeline/ci.yml

```diff
@@ -378,6 +378,7 @@ jobs:
 
 - job: 'LinuxMakeGNU_OpenBLAS_x86'
```

Comment:
Why not remove this entry altogether instead?

Reply:
Done.